index.html

```diff
@@ -0,0 +1,48 @@
+<!DOCTYPE html>
+<html lang="en" dir="ltr">
```

Comment:
What is dir ?

Reply:
The dir attribute specifies the text direction of the element's content.
there are two type of dir
ltr - means left-to-right text direction
rtl - means right to left text direction